framework_tvb/tvb/core/data_encryption_handler.py

```diff
@@ -0,0 +1,193 @@
+# -*- coding: utf-8 -*-
```

Comment:
could this module stay under `services` or in `core.entities.file`?

Reply:
I moved it under services